stdlib/public/Platform/MachError.swift

```diff
@@ -12,7 +12,9 @@
 
 #if os(OSX) || os(iOS) || os(tvOS) || os(watchOS)
 /// Enumeration describing Mach error codes.
-@objc public enum MachErrorCode : Int32 {
+@_fixed_layout // FIXME(sil-serialize-all)
+@objc
+public enum MachErrorCode : Int32 {
```

Comment:
Probably removing the annotation from errors is one of the first changes you can try

Reply:
Certainly `@objc` enums shouldn't be affected.